### PR TITLE
Fix/rename variables

### PR DESCRIPTION
**Pull request by apenzk**
# Description of change
Rename some variables to be more accurate
## Type of change
- Bug fix (a non-breaking change which fixes an issue)
## How the change has been tested
This is just a rewrite of names.

### CODE REVIEW

File: plugins/messagelayer/parameters.go [hunk 1]
```diff
@@ -18,9 +18,9 @@ var Parameters = struct {
 		GenesisNode string `default:"Gm7W191NDnqyF7KJycZqK7V6ENLwqxTwoKQN4SmpkB24" usage:"the node (base58 public key) that is allowed to attach to the genesis message"`
 	}
 
-	// FCOB contains parameters related to the fast consensus of barcelona.
+	// FCOB contains parameters related to the FPC quarantine time.
```
Comment (apenzk) [inline plugins/messagelayer/parameters.go:21]:
```suggestion
	// FCOB contains parameters related to the transaction quarantine time before applying (if necessary) FPC.
```

File: plugins/messagelayer/plugin.go [hunk 2]
```diff
@@ -22,8 +22,8 @@ import (
 )
 
 const (
-	// DefaultAverageNetworkDelay contains the default average time it takes for a network to propagate through gossip.
-	DefaultAverageNetworkDelay = 5 * time.Second
+	// DefaultUpperBoundNetworkDelay contains the default upper bound duration it takes for a message to propagate through the network through gossip.
+	DefaultUpperBoundNetworkDelay = 5 * time.Second
```
Comment (jonastheis) [inline plugins/messagelayer/plugin.go:26]: Is that used anywhere except the integration tests?

Comment (apenzk) [reply plugins/messagelayer/plugin.go:26]: nope

**Review by capossele — APPROVED** [review]

File: plugins/messagelayer/plugin.go [hunk 3]
```diff
@@ -120,8 +120,8 @@ func configure(plugin *node.Plugin) {
 		plugin.LogInfof("read snapshot from %s", Parameters.Snapshot.File)
 	}
 
-	fcob.LikedThreshold = time.Duration(Parameters.FCOB.AverageNetworkDelay) * time.Second
-	fcob.LocallyFinalizedThreshold = time.Duration(Parameters.FCOB.AverageNetworkDelay*2) * time.Second
+	fcob.LikedThreshold = time.Duration(Parameters.FCOB.QuarantineTime) * time.Second
+	fcob.LocallyFinalizedThreshold = time.Duration(Parameters.FCOB.QuarantineTime*2) * time.Second
```
[the remark this answers is not in the document]
Comment (capossele) [reply plugins/messagelayer/plugin.go:124]: Can you fix this?

File: plugins/messagelayer/plugin.go (hunk 2 above)
Comment (apenzk) [reply plugins/messagelayer/plugin.go]: moved it to parameters, directly behind the FCOB rule parameter

**Pull request merged**